RELEASE-NOTES.txt

```diff
@@ -1,4 +1,5 @@
 Unreleased
+* [**] Add fullscreen embed preview to Android [https://github.com/WordPress/gutenberg/pull/50440]
```

Comment:
I'm wondering if this entry could point to this PR instead of the Gutenberg one, WDYT?